cf-editor-form-field-customization-sample/hooks/build-static.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+
+const exists = async (path) => {
+  try {
+    await fs.promises.access(path);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+module.exports = async (config) => {
+  const dist = config.web.distProd;
+
+  if (!(await exists(dist))) {
+    await fs.promises.mkdir(dist, { recursive: true });
+  }
+  if (!(await exists(dist + "/tinymce.min.js"))) {
```

Comment:
Maybe better to read once what the files folder contains and check programmatically if they exist instead of each time calling the filesystem?

PS. Can u please explain in general why u need to copy the package from node_modules?

Reply:
@VladimirZaets ,
> Can u please explain in general why u need to copy the package from node_modules?

The TinyMCE files need to be hosted somewhere, and by copying them to the `dist` directory, they will be placed on Adobe CDN. See https://www.tiny.cloud/docs/tinymce/6/react-pm-host/#procedure , "4. Setup a postinstall script to copy TinyMCE to the public directory for hosting" for more details.

Reply:
@VladimirZaets , I am checking the existence of the _high-level_ TinyMCE-related directories, which, in their turn, contain multiple sub-directories. I'm not sure if I can optimize it in terms of execution time.

Reply:
@slitviachenko Have you investigated if lazy loading works in this case?
See https://react.dev/reference/react/lazy for more details

Reply:
@irenelagno , Could you please elaborate more on how the code can benefit from it? We need to substitute RTE fields with our custom one during a CF Editor opening.